testing/xfpga/test_properties_c.cpp

```diff
@@ -53,8 +58,8 @@ class properties_c_p : public ::testing::TestWithParam<std::string> {
     system_->prepare_syfs(platform_);
     ASSERT_EQ(xfpga_plugin_initialize(), FPGA_OK);
     ASSERT_EQ(xfpga_fpgaGetProperties(nullptr, &filter_accel_), FPGA_OK);
-    ASSERT_EQ(fpgaPropertiesSetDeviceID(filter_accel_, 
-                                        platform_.devices[0].device_id), FPGA_OK);
+    auto accel_dev_id = platform_.devices[0].device_id + platform_.devices[0].num_vfs;
```

Comment:
Not convinced this will always be true.

Reply:
You're right. This will work for now but fixing this the right way would require more work to the testing hw tables and data structures.